Core/Src/spi_command_handler.c

```diff
@@ -130,12 +134,23 @@ int spi_handle_command(uint8_t obc_cmd) {
         uint8_t buffer[sizeof(hk)];
         memcpy(buffer, &hk, sizeof(hk));
         spi_transmit(buffer, sizeof(buffer));
-
         return 0;
     }
     case IRIS_TAKE_PIC: {
         // needs dedicated thought put towards implement
-        arducam_capture_image(0);
+        arducam_set_resolution(JPEG, 640, VIS);
```

Comment:
Not a great place for setting the resolution. That should be done on boot. If we want timing accuracy we shouldn't be writing a flurry of I2C regs when we should be taking a picture

Reply:
Agreed, I will make sure that all sensor reg information is pre-defined during boot